src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountSmall = 20;
+  const discountBig = 50;
+  let costRent = 40 * days;
```

Comment:
It would be great to create a variable for 40 too.
Also, you can make it a const now